nemo/collections/nlp/data/token_classification/token_classification_dataset.py

```diff
@@ -87,6 +87,11 @@ def get_features(
 
         for j, word in enumerate(words):
             word_tokens = tokenizer.text_to_tokens(word)
+
+            # to handle emojis that could be neglected during tokenization
+            if len(word.strip()) > 0 and len(word_tokens) == 0:
+                word_tokens = [tokenizer.ids_to_tokens(tokenizer.unk_id)]
```

Comment:
Same as above. Basically we need multiple unk tokens - 1 for every emoji (even if they are consecutive)

Reply:
see the above